### ISSUE TITLE
Peer dependency issue with React 19

### ISSUE THREAD

**Issue by LuckyT89** [immerjs/use-immer#140]
Next.js recently released version 15. This now includes React 19 when creating a new Next.js project. After creating a new project and trying to install the use-immer package there is now an error message relating to peer dependencies, basically saying it is looking for the following versions of React: `^16.8.0 || ^17.0.1 || ^18.0.0` but it instead found version 19. I was able to install the package using the `--lecagy-peer-deps` flag but it does not seem to be working properly. 

Is there a timeframe for when the peer dependencies for use-immer will be updated to include React 19?

**Comment by yanglee2421** [comment]
#141 This might solve it

**Comment by sumannath** [comment]
`npm install use-immer` still uses the old version and throws the same error though the fix has been merged to master 2 days ago

**Comment by LuckyT89** [comment]
It looks like a new release still needs to be made. It seems npm is using version 0.10.0 which was created on June 10.

**Comment by mweststrate** [comment]
Yeah, this repo has so few updates that I've never bothered to set up an automated release process :) Just released 0.11

**Comment by LuckyT89** [comment]
I just tested on a new project using React 19 and I can now install use-immer and use it without any issue. Thank you!
